src/energy_comms/transform/msha.py

```diff
@@ -0,0 +1,41 @@
+"""Transform functions for the MSHA mine data."""
+from typing import Literal
+
+import geopandas
+import pandas as pd
+
+import energy_comms
+
+
+def transform(
+    raw_df: pd.DataFrame, census_geometry: Literal["state", "county", "tract"] = "tract"
+) -> pd.DataFrame:
+    """Standardize columns, filter for IRA coal mine criteria, join to census tract."""
+    df = raw_df.copy()
+    df.columns = df.columns.str.lower()
+    df["current_status_dt"] = pd.to_datetime(df["current_status_dt"].astype("string"))
+    # apply filters for IRA criteria
+    mask = (
+        (df.current_mine_status.isin(["Abandoned and Sealed", "Abandoned"]))
```

Comment:
I think I'll add in Non-Producing as well

Reply:
That sounds great — I think we could split the mine categories into 2 distinct ones. The first being the ones already included in the code (e.g. abandoned and sealed) that follow the legislative text strictly and the second with non-producing, temporarily idled, and other criteria we feel is more expansive.